config/initializers/doxxer.rb

```diff
@@ -0,0 +1,12 @@
+class Doxxer
+  BASE_URL = 'https://just-arrived.herokuapp.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.